index.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css" integrity="sha512-xh6O/CkQoPOWDdYTDqeRdPCVd1SpvCA9XXcUnZS2FmJNp1coAFzvtCN9BmamE+4aHK8yyUHUSCcJHgXloTyT2A==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+  </head>
+
+  <body>
+    <nav id="navbar">
+      <ul class="nav-list">
+        <li><a href="#about">About</a></li>
+        <li><a href="#work">Work</a></li>
+        <li><a href="#contact">Contact</a></li>
+      </ul>
+    </nav>
```

Comment:
Surround the ```nav``` tag with the ```header``` tag, it is a better semantic tag.

Reply:
- [ ] Hi @martinkarugaba, Good job :+1:  for using [GitHub inline comments feature](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) and provide for reference for suggested changes, But please kindly makes sure that you phrased the comments politely, as suggestions instead of orders and includes an explanation, a "why", for each requested change.

for example

- [ ] I think it would be better if you surround or wrap the `nav` tag with the `header` tag, it gives your page better semantics. Find out more about HTML semantics here [HTML Semantics](https://github.com/microverseinc/curriculum-html-css/blob/main/html5.md).

Reply:
- Thank you for the review @Owusu-Desmond, I'll make sure I keep that in mind the next time I'm giving a comment.